dotnet/src/Connectors/Connectors.MistralAI/ChatCompletion/MistralAIChatCompletionService.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Http;
+using Microsoft.SemanticKernel.Services;
+
+namespace Microsoft.SemanticKernel.Connectors.MistralAI;
+
+/// <summary>
+/// Mistral chat completion service.
+/// </summary>
+public sealed class MistralAIChatCompletionService : IChatCompletionService
```

Comment:
Are we not implementing text completion as well as with OpenAI?

Reply:
MistalAI doesn't support text completion and even OpenAI has moved away from this.